components/supervisor/cmd/rings.go

```diff
@@ -208,7 +208,7 @@ var ring1Cmd = &cobra.Command{
 		// (cw) I have been able to reproduce this issue without newuidmap/newgidmap.
 		//      See https://gist.github.com/csweichel/3fc9d4b0752367d4a436f969c8685c06
 		runtime.LockOSThread()
-		err = unix.Prctl(unix.PR_SET_PDEATHSIG, uintptr(unix.SIGKILL), 0, 0, 0)
+		unix.Prctl(unix.PR_SET_PDEATHSIG, uintptr(unix.SIGKILL), 0, 0, 0)
```

Comment:
this lints with errcheck?

Reply:
Apparently yes
![image](https://user-images.githubusercontent.com/24193764/103008908-9bc56700-4514-11eb-8744-68bf4ee44445.png)